src/components/common/EffectiveDateTime.vue

```diff
@@ -102,12 +102,8 @@ export default class EffectiveDateTime extends Mixins(DateMixin) {
     minuteSelector: FormFieldType // used in unit tests
   }
 
-  /** Current JS date, expected to be passed in periodically. */
-  @Prop() readonly currentJsDate: Date
-
-  /** Effective Date Time object, for initial config. */
-  @Prop() readonly effectiveDateTime: EffectiveDateTimeIF
-
+  @Getter getCurrentJsDate!: Date
+  @Getter getEffectiveDateTime!: EffectiveDateTimeIF
```

Comment:
I replaced the props with getters since this component already used store getters.